tests/bench/oolong.py

```
@@ -0,0 +1,763 @@
+            base_url=honcho_url,
+        )
+
+    async def wait_for_deriver_queue_empty(
+        self, honcho_client: AsyncHoncho, session_id: str | None = None
+    ) -> bool:
+        """Wait for deriver queue to be empty."""
+        start_time = time.time()
+        while True:
+            try:
+                status = await honcho_client.get_deriver_status(session=session_id)
+            except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)